src/asm/lexer.c

```diff
@@ -47,9 +47,12 @@
 
 /* Neither MSVC nor MinGW provide `mmap` */
 #if defined(_MSC_VER) || defined(__MINGW32__)
-# include <windows.h>
-# include <fileapi.h>
-# include <winbase.h>
+# define WIN32_LEAN_AND_MEAN
```

Comment:
What does this do?

Reply:
`WIN32_LEAN_AND_MEAN` makes `windows.h` include less. Something it was including without that was causing a build error. (I don't remember exactly what, I was doing a lot of force-pushes attempting to not include windows.h at all before finding that worked.)